content/projects/gsoc/index.adoc

```diff
@@ -27,8 +27,8 @@ image:/images/gsoc/opengraph.png[Jenkins GSoC, role=center, float=center]
 
 == GSoC 2023
 
-We have applied to participate in Google Summer of Code 2023. 
-See the (link:https://docs.google.com/document/d/1FYOBo12qz24Vxq0TxWuv9ElHH_rHP51ouMsPms4tTmw/edit?usp=sharing[Jenkins GSoC Organisation Application Form]).
+We have been accepted as a Google Summer of Code mentoring org for 2023. +
```

Comment:
Is there any significance to the plus sign `+` at the end of this line?

Reply:
This is to add a <br/> at the end of the line. It is more readable IMHO

Reply:
Okay, I see 👍🏼